test/errors/unsupportedAsIdent1-detailed.good

```diff
@@ -0,0 +1,8 @@
+─── error in unsupportedAsIdent1.chpl:6 [UnsupportedAsIdent] ───
+  This form of 'as' is not yet supported.
+  'as' requires both the original and new names to be simple identifiers, but instead got the following:
```

Comment:
The "but" seems a tiny bit awkward here, but I don't have concrete ideas on how to fix it.

Reply:
I'm a fan of semicolons, though I know a lot of others aren't, but I would do: `'as' requires both the original and new names to be simple identifiers; instead got the following:`

Reply:
Actually, now that I look at it implemented, I think that's worse than the "but". Hm...